### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by naji-97**
1. Add the Online Version Link to the Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 
-For this repository, I've created html templates and styling files in order to built a responsive portfolio , which can help to easy explorate profiles from differents devices.
+For this repository, I've created html templates and styling files in order to built a responsive portfolio, which can help to easy explorate profiles from differents devices.
 
 ![Live Demo Link](<https://github.com/naji-97/Portfolio-/assest/img/Screenshot> 2022-12-02 111444.png/)
```
Comment (leonardodiasb) [inline README.md]: - [ ] Unfortunately, I could not open the page with the link you added to your README file. This link looks like an image you had on your repository. Please add the correct link to the deployed app on GitHub Pages to your README. Kindly pay extra attention to the formatting in the README file.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @naji-97, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ For this repository, I've created html templates and styling files in order to b
 [Online Version Link](<https://naji-97.github.io/Portfolio-/)
 
 ## Live Demo
-[Live Demo Link](<https://github.com/naji-97/Portfolio-/assest/img/Screenshot> 2022-12-02 111444.png/)
+[Live Demo Link](<https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=t79ZHc8kYVy3uIUT-0)
```
Comment (franclobo) [inline README.md:33]: - [ ] **[OPTIONAL]** KIndly replace this link with the [online version link,](https://naji-97.github.io/Portfolio-/) it is confusing to the user and the code reviewer when using the live demo link additionally the [live demo link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=t79ZHc8kYVy3uIUT-0) returns the 404 error.

**Review by franclobo — APPROVED** [review]
Hi @naji-97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The link works as expected ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 
- No linter errors ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (naji-97) [reply README.md:33]: Thanks

**Pull request merged**